frontend/pages/LogoutPage/LogoutPage.tsx

```diff
@@ -0,0 +1,12 @@
+import { useDispatch } from "react-redux";
+// @ts-ignore
+import { logoutUser } from "../../redux/nodes/auth/actions";
+
+const LogoutPage = (): boolean => {
+  const dispatch = useDispatch();
+
+  dispatch(logoutUser());
```

Comment:
Are we returning too soon here? Can we await the logout?

Reply:
I'm not even sure what the return is doing. To control this better, I think using `useEffect` is the way to go. So something like 
```
useEffect(() => {
  dispatch(logoutUser());
}, [])
```

and use `return null;` signifying that the component renders nothing.

Reply:
@RachelElysia I think I may have found the issue (inside redux of course). LMK if you would like to hop on a call to try to hammer it out.